Orange/widgets/data/owoutliers.py

```diff
@@ -317,32 +338,12 @@ def send_report(self):
                            ("Inliers", self.n_inliers),
                            ("Outliers", self.n_outliers)))
 
-        params = self.current_editor.get_parameters()
-        if self.outlier_method == self.OneClassSVM:
-            self.report_items(
-                "Detection",
-                (("Detection method",
-                  "One class SVM with non-linear kernel (RBF)"),
-                 ("Regularization (nu)", params["nu"]),
-                 ("Kernel coefficient", params["gamma"])))
-        elif self.outlier_method == self.Covariance:
-            self.report_items(
-                "Detection",
-                (("Detection method", "Covariance estimator"),
-                 ("Contamination", params["contamination"]),
-                 ("Support fraction", params["support_fraction"])))
-        elif self.outlier_method == self.LOF:
-            self.report_items(
-                "Detection",
-                (("Detection method", "Local Outlier Factor"),
-                 ("Contamination", params["contamination"]),
-                 ("Number of neighbors", params["n_neighbors"]),
-                 ("Metric", params["metric"])))
-        elif self.outlier_method == self.IsolationForest:
-            self.report_items(
-                "Detection",
-                (("Detection method", "Isolation Forest"),
-                 ("Contamination", params["contamination"])))
+        if (self.outlier_method == self.OneClassSVM) or \
+                (self.outlier_method == self.Covariance) or \
+                (self.outlier_method == self.LOF) or \
+                (self.outlier_method == self.IsolationForest):
```

Comment:
You can remove this `if` and `else` (together with `raise`).

I think that @VesnaT wanted to raise an exception is somebody adds a new method and forgets to add it to report. But now that you implemented `get_report_parameters` this test is no longer necessary. If somebody adds a new editor and forgets to implement `get_report_parameters`, the base class' will raise this exception. So just calling `self.current_editor.get_report_parameters` suffices.

(By the way, a more Pythonic way for this expression is `if self.outlier_method in (self.OneClassSVM, self.Covariance, self.LOF, self.IsolationForest)`.)